### PR TITLE
Update teams page

### PR DESCRIPTION
**Pull request by Tanza3D**
diff for this one is scuffed

### CODE REVIEW

File: home/api/team.php [hunk 1]
```diff
@@ -115,11 +113,25 @@ function cleanTeam()
     ]
 );
 addMember(
-    14125695,
-    "TheEggo",
-    "??home.team.role.generic.snapshotsDeveloper??",
+    10238680,
+    "chromb",
+    "chromb",
+    "chromb",
```
Comment (minusQuantumNeko) [inline home/api/team.php:119]: Lmfao

Comment (Tanza3D) [reply home/api/team.php:119]: chromb

**Review by EXtremeExploit — APPROVED** [review]

File: home/api/team.php [hunk 2]
```diff
@@ -173,6 +180,17 @@ function cleanTeam()
     null,
     []
 );
+addMember(
+    1699875,
+    "Remyria",
+    "Alumni",
```
Comment (minusQuantumNeko) [inline home/api/team.php:186]: Maybe worth adding his former position too? For example `Alumni (Community Manager)`

**Comment by Tanza3D** [comment]
pedrito already merged it lol

**Review by minusQuantumNeko — CHANGES_REQUESTED** [review]
Add my website and Bluesky accounts and maybe consider what I suggested for Remyria

**Pull request merged**